files/initramfs-tools/fsck-rootfs

```diff
@@ -11,15 +11,16 @@ root_val=""
 set -- $(cat /proc/cmdline)
 for x in "$@"; do
     case "$x" in
-        root=*)
-            root_val="${x#root=}"
+        root=UUID=*)
+            root_val="${x#root=UUID=}"
```

Comment:
This makes it only handle cases where the UUID is passed in for the root device, right? What about when the device name is used instead? I think some devices have the device name instead of the UUID for whatever reason.

Reply:
@saiarcot895 addressed now